Fix paths in data_paths.yaml and dcmip6_all_paths.yaml following storage migration

brews opened the pull request:
Changes hard-coded paths from Iowa buckets to Oregon buckets. Some ad-hoc notebook analyses use these files. The files is unusable without this update.

This was a basic "find-all/replace" on the file, replacing the stale buckets.

Related to #640.

brews commented:
I think @kemccusker @dgergel will run into trouble rerunning notebook analysis without this update so putting you two as reviewers. Any reason we shouldn't follow through with this update now?

dgergel commented:
Thanks @brews - I've already added a find/replace to several workflows to deal with this but that should be fine even if the data paths are updated. 

Some notebooks for the paper use this yaml: `dcmip6_all_paths.yaml` - could you run the same find/replace on that one as well?

kemccusker commented:
Awesome, thanks @brews ! I was planning to use the Oregon trail mapping dict to solve this but this is better. Thank you!

brews commented:
Absolutely, @dgergel. I just updated that file with [540faa6](https://github.com/ClimateImpactLab/downscaleCMIP6/pull/654/commits/540faa6ab5cd0a584af805a99d8f4a86c402038f).

Updated the description and such to match this addition.
